packages/apps/src/Apps.tsx

```diff
@@ -3,7 +3,7 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import { BareProps } from '@polkadot/ui-app/types';
-import { SideBarTransition } from '@polkadot/ui-app/constants';
+import { SideBarTransition, SIDEBAR_TRANSITION_DURATION, SIDEBAR_MENU_THRESHOLD } from '@polkadot/ui-app/constants';
```

Comment:
I'd still move them to a new file `./apps/src/constants.ts` because they are all just used in this app, not in `./ui-app`

Reply:
Agreed. If we start using them everywhere, we can move them to ui-app

Reply:
Ok, keeping the compartmentalisation consistency.